packages/skills/weather_station_ledger/strategy.py

```diff
@@ -86,7 +86,7 @@ def generate_proposal_and_data(self, query: Query) -> Tuple[Description, Dict[st
         total_price = self.price_per_row * rows
         proposal = Description({"rows": rows,
                                 "price": total_price,
-                                "currency_pbk": self.currency})
+                                "currency_pbk": self.currency[1]})
```

Comment:
In the configs for strategy specify the `ledger_id` and then load it in the constructor and choose accordingly.